app/views/early_career_payments/claims/eligible_degree_subject.html.erb

```diff
@@ -0,0 +1,49 @@
+<% claim = current_claim.for_policy(LevellingUpPremiumPayments) %>
+
+<% content_for(:page_title, page_title(t("early_career_payments.questions.eligible_degree_subject"), policy: current_policy_routing_name, show_error: claim.errors.any?)) %>
+
+<% path_for_form = claim.persisted? ? claim_path(current_policy_routing_name) : claims_path(current_policy_routing_name) %>
```

Comment:
Maybe for another PR, but isn't this always persisted? Check with @nigel-lowry maybe?

Reply:
I blindly copied this across, it seems to be in every view. @nigel-lowry Any input on this?

Reply:
Before I moved the `current-school` slug to the start of the wizard (and created the record manually in the claims controller) this was necessary just in case a certain slug was ever moved to the start of the wizard (this wouldn't have worked with the `current-school` which is why I had to do a lot of work for that ticket).

I'm pretty sure you can get rid of all of those now (but probably only for the ECP/LUP journey) but I never actually tried it.